x/qgb/orchestrator/orchestrator_client.go

```diff
@@ -3,13 +3,12 @@ package orchestrator
 import (
 	"context"
 	"fmt"
-	"sync"
-
 	"github.com/celestiaorg/celestia-app/x/qgb/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	tmlog "github.com/tendermint/tendermint/libs/log"
 	"github.com/tendermint/tendermint/rpc/client/http"
 	coretypes "github.com/tendermint/tendermint/types"
+	"sync"
```

Comment:
Why was this moved here? It looks like other files separate standard from cosmos/celestia dependencies with a newline (as did this file before this PR). We should maintain a consistent style here, ideally enforced by CI.